lib/criticality/functions.dart

```diff
@@ -5,7 +5,6 @@ import "dart:math";
 ///returns null if any parameter is null
 double? rpnFunc(double? system, int? freq, int? impact) {
   try {
-    print('{system: $system, freq: $freq, impact: $impact}');
     return system! * freq! * impact!;
```

Comment:
judging by number of errors "error: Null check operator used on a null value"
this is calculating for every row everytime a cell changes.
will need to be fixed